.gitignore

```diff
@@ -46,3 +46,4 @@ settings_local.py
 /sandbox/public/
 /sandbox/whoosh_index/
 /sandbox/logs/
+/oscar
```

Comment:
Why are you adding this to .gitignore?

Reply:
oh, sorry, it's a mistake, it's my virtualenv folder name, you can just delete it.